packages/custom-properties/package.json

```diff
@@ -19,7 +19,7 @@
     "pluralize": "^8.0.0"
   },
   "devDependencies": {
-    "@theme-ui/core": "^0.3.1",
-    "@types/pluralize": "^0.0.29"
+    "@theme-ui/css": "^0.3.1",
+    "@types/pluralize": "0.0.29"
```

Comment:
You accidentally pinned the dependency here

Reply:
damn, thanks! pinning `types/pluralize` makes totally no sense here.